Update announcements.md (#4291)
camunda/camunda-docs: docs/reference/announcements.md

Camunda 8 Self-Managed clusters are not affected by this.

### Support for Amazon OpenSearch for Optimize

> This release supports an extended feature set in Optimize that is available on OpenSearch. Full support is committed for the next release in Jan 2025.

Reviewer: Is the referenced feature set in OpenSearch (and we now support it), or in Optimize (and it uses OpenSearch)? It's a little unclear. Do we have any additional pages we can link to for more information/to clarify?

Author: @conceptualshark, there are no additional pages I can think of. This announcement is similar to the one we used for previous releases. What are you looking exactly for? A list of features that are supported on Optimize?

Reviewer: ```suggestion
This release extends the OpenSearch features supported by Optimize. Full support is committed for the next release in January 2025.
```

^ Is this accurate? It would be a little clearer if so. No need for links if we don't have anything supporting.

Author: Yes - that sounds good.
